src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -312,6 +312,9 @@ public void handleMessage(Message msg) {
                     case MESSAGE_HANDLE_EDITOR_BINDINGS_RECEIVED:
                         handleEditorBindingsReceived((JSONObject) msg.obj);
                         break;
+                    case MESSAGE_HANDLE_EDITOR_CHANGES_CLEARED:
```

Comment:
It's per change- the object's actions are a list of change ids that we use to undo (all changes get cleared on disconnect in a separate mechanism that doesn't require the editor to exist)